docs/core/tools/elevated-access.md

```diff
@@ -0,0 +1,182 @@
+
+### Run the global tool
+
+**Option 1** Use full path with sudo:
+
+```bash
+sudo /usr/local/share/dotnet-tools/TOOLCOMMAND
+```
+
+**Option 2** Add symbol link of the tool, once per tool:
+
+```bash
+sudo ln -s /usr/local/share/dotnet-tools/TOOLCOMMAND /usr/local/bin/TOOLCOMMAND
+```
+
+And run with:
+
+```bash
+sudo TOOLCOMMAND
+```
+
+### Uninstall the global tool
+
+```bash
+sudo dotnet tool uninstall PACKAGEID --tool-path /usr/local/share/dotnet-tools
+```
+
+If you made a symbol link, you also need to remove it:
+
+```bash
+sudo rm /usr/local/bin/TOOLCOMMAND
+```
+
+# [macOS](#tab/macos)
+
+### Install the global tool
+
+Package assets should be installed in a protected location using the `--tool-path` option. This separation avoids sharing a restricted user environment with an elevated environment.
+
+```bash
+sudo dotnet tool install PACKAGEID --tool-path /usr/local/share/dotnet-tools
+```
+
+`/usr/local/share/dotnet-tools` will be created with permission `drwxr-xr-x`. If the directory already exists, use the `ls -l` command to verify the restricted user doesn't have permission to edit the directory. If so, use `sudo chmod o-w -R /usr/share/dotnet-tools` command to remove the access.
+
+### Run the global tool
+
+**Option 1** Use full path with sudo:
+
+```bash
+sudo /usr/local/share/dotnet-tools/TOOLCOMMAND
+```
+
+**Option 2** Add symbol link of the tool, once per tool:
+
+```bash
+sudo ln -s /usr/local/share/dotnet-tools/TOOLCOMMAND /usr/local/bin/TOOLCOMMAND
+```
+
+And run with:
+
+```bash
+sudo TOOLCOMMAND
+```
+
+### Uninstall the global tool
+
+```bash
+sudo dotnet tool uninstall PACKAGEID --tool-path /usr/local/share/dotnet-tools
+```
+
+If you made a symbol link, you also need to remove it:
+
+```bash
+sudo rm /usr/local/bin/TOOLCOMMAND
+```
+
+---
+
+## Local tools
+
+Local tools are scoped per subdirectory tree, per user. When run elevated, local tools share a restricted user environment to the elevated environment. In Linux and macOS, it results in files being set with root user only access. If the user switch back to a restricted account, the user can no longer access or write to the files. So, installing tools that require elevation as local tools isn't recommended. Instead, use the `--tool-path` option and the previous guidelines for global tools.
+
+## Elevation during development
+
+During development, you may need elevated access to test your application. This scenario is common for IoT apps, for example. We recommend that you build the application without elevation and then run it with elevation. There are a few patterns, as follows:
+
+- Using generated executable (it provides the best startup performance):
+
+```bash
+dotnet build
+sudo ./bin/Debug/netcoreapp3.0/APPLICATIONNAME
+```
+
+Using the [dotnet run](dotnet-run.md) command with the `—no-build` flag to avoid generating new binaries:
```

Comment:
This should be a bulleted list item like the previous one, and the code indented.